akka-persistence/src/main/scala/akka/persistence/fsm/PersistentFSM.scala

```diff
@@ -54,6 +54,19 @@ trait PersistentFSM[S <: FSMState, D, E] extends PersistentActor with Persistent
   private var currentStateTimeout: Option[FiniteDuration] = None
 
   /**
+   * "akka.persistence.fsm.snapshot-after"
+   */
+  private val snapshotAfter: Option[Int] = {
+    val config = context.system.settings.config
+    val key = "akka.persistence.fsm.snapshot-after"
+
+    config.getString(key).toLowerCase match {
+      case "off" ⇒ None
+      case _     ⇒ Some(config.getInt(key))
+    }
+  }
```

Comment:
thanks, I think this http://doc.akka.io/docs/akka/2.5/scala/extending-akka.html#Application_specific_settings is good example of writing such a "configuration" extension